extensions/blocks/calendly/edit.js

```diff
@@ -250,8 +250,10 @@ export default function CalendlyEdit( { attributes, className, clientId, setAttr
 		</InspectorControls>
 	);
 
+	const classes = `${ className } calendly-style-${ style }`;
```

Comment:
You may have to switch to `getBlockDefaultClassName` like you did in #14725?

Reply:
I think `className` is good here as that means we'll have any additional classes that have been added in the advanced settings, and the `calendly-style-` class won't be affected by that issue.